templates/shared/inventory2.hbs

```diff
@@ -14,10 +14,10 @@
                     <span class="separator">&sol;</span>
                     <span class="max">{{ max }}</span>
                 </div>
-                <i class="breakpoint encumbrance-33 arrow-up" role="presentation"></i>
-                <i class="breakpoint encumbrance-33 arrow-down" role="presentation"></i>
-                <i class="breakpoint encumbrance-66 arrow-up" role="presentation"></i>
-                <i class="breakpoint encumbrance-66 arrow-down" role="presentation"></i>
+                <i class="breakpoint encumbrance-low arrow-up" role="presentation" style="--encumbrance-low: {{ low }}%"></i>
+                <i class="breakpoint encumbrance-low  arrow-down" role="presentation" style="--encumbrance-low: {{ low }}%"></i>
+                <i class="breakpoint encumbrance-high arrow-up" role="presentation" style="--encumbrance-high: {{ high }}%"></i>
+                <i class="breakpoint encumbrance-high arrow-down" role="presentation" style="--encumbrance-high: {{ high }}%"></i>
```

Comment:
Set these above above on the meter div to avoid setting them twice:

```hbs
            <div class="meter progress" role="meter" aria-valuemin="0" aria-valuenow="{{ pct }}"
                 aria-valuetext="{{ value }}" aria-valuemax="{{ max }}" style="--bar-percentage: {{ pct }}%;
                 --encumbrance-low: {{ stops.encumbered }}%; --encumbrance-high: {{ stops.heavilyEncumbered }}%;">
```

```suggestion
                <i class="breakpoint encumbrance-low arrow-up" role="presentation"></i>
                <i class="breakpoint encumbrance-low  arrow-down" role="presentation"></i>
                <i class="breakpoint encumbrance-high arrow-up" role="presentation"></i>
                <i class="breakpoint encumbrance-high arrow-down" role="presentation"></i>
```

Reply:
I've moved this as suggested. Only thing different is I've put the style string all on one line as otherwise it gets a line break in the HTML and looks messy from that side. I had a look but I couldn't find a fancy handlerbar method to remove line breaks.

Reply:
If you do it within a handlebars element, you can use tilde:

```hbs
<div>   {{~ localize "blah" }}</div> <!-- Removes any space before -->
<div>{{ localize "blah" ~}}   </div> <!-- Removes any space after -->
<div>   {{~ localize "blah" ~}}   </div> <!-- Removes space on both sides -->
```